nomenclature/definitions/variable/fuel_types.yaml

```diff
@@ -0,0 +1,80 @@
+# This list of fuels will replace any occurrence of `<Fuel>`
+# in a variable-definition yaml file
+
+# All fuels that have the attribute "ccs: True" have two sub-categories
+# "w/ CCS": with a CO2 capture component
+# "w/o CCS": with freely vented CO2 emissions
+
+<Fuel>:
+  Biomass:
+    description: Purpose-grown biomass, crop residues, forest industry waste,
+                 municipal solid waste
+    ccs: True
+
+  Coal:
+    description: Coal (including hard coal and lignite)
+    ccs: True
+
+  Fossil:
+    description: All fossil fuels (crude oil, coal, natural gas)
+    ccs: True
+
+  Gas:
+    description: Natural gas (including methane from biomass or hydrogenation)
+    ccs: True
+
```

Comment:
Gas|OCGT:
    description: Natural gas (including methane from biomass or hydrogenation). A typical Open Cycle Gas Turbine power plant.
    ccs: True

  Gas|CCGT:
    description: Natural gas (including methane from biomass or hydrogenation). A typical Combined Cycle Gas Turbine power plant. It delivers additional electricity through a heat recovery system that captures exhaust heat from the gas turbine.
    ccs: True

Reply:
merged via suggestion by @sandrinecharousset